src/pages/resources.js

```diff
@@ -1,12 +1,28 @@
 import React from "react";
 import DefaultLayout from "../components/layouts/DefaultLayout";
 import SEO from "../components/seo";
-import { Heading } from "@chakra-ui/core";
+import { Heading, Text } from "@chakra-ui/core";
+import LinksTable from '../components/LinksTable';
+import linksTableData from '../utils/linksTableData';
+import newsLinksTableData from '../utils/newsLinksTableData';
 
 const Resources = () => (
   <DefaultLayout>
     <SEO title="Resources" />
     <Heading as="h1">Resources</Heading>
+    <Text fontSize="xl">
+      Use the resources below to learn more about technology responses to COVID-19, digital contact tracing, exposure notification, GPS, Bluetooth, and other technology and policy issues facing public health authorities and private sector organizations.
+    </Text>
+
+    {linksTableData.map((data, key) => (
```

Comment:
I see where you're going with this by extracting the data into its own file... but let's keep both of those data objects internal for the time being. This is a temporary thing so don't want to create the extra layers for now.